extensions/blocks/social-previews/editor.js

```diff
@@ -1,7 +1,31 @@
 /**
  * Internal dependencies
  */
-import { name, settings } from '.';
+import { name, settings, SocialPreviews } from '.';
 import registerJetpackPlugin from '../../shared/register-jetpack-plugin';
+import { registerPlugin } from '@wordpress/plugins';
+import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
 
+// Register the main "social-previews" extension if the feature is available
+// on the current plan.
 registerJetpackPlugin( name, settings );
+
+// TODO: remove in favour of `getJetpackExtensionAvailability()` call.
+// Testing purposes only to force reveal of upgrade nudge UI.
+const extensionAvailableOnPlan = getJetpackExtensionAvailability( 'social-previews' );
+// const extensionAvailableOnPlan = false;
```

Comment:
Let's cleanup those todo comments and testing variant and keep just the what's used